discord_music_bot/commands/queue.py

```diff
@@ -69,8 +69,18 @@ async def clear_queue(self, ctx: discord.ApplicationContext) -> None:
     )
     async def play_next(self, ctx: discord.ApplicationContext) -> None:
         player = await get_current_player(ctx)
-
-        # .stop() will trigger on_wavelink_track_end event
-        # that automatically starts playing next track
-        await player.stop()
-        await ctx.respond("Skipping current track...")
+        skipped_item_title = player.track.title
+
+        if not player.queue.is_empty:
```

Comment:
Consider inverting the `if...else`.
If the queue is empty, you stop playback and return, otherwise play next track.